### PR TITLE
[1LP][RFR] provision tests/views fixes

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
1. Fixed template select in provision table
2. Added BZ blocker for 5.10

{{pytest: cfme/tests/cloud_infra_common/test_provisioning.py -k test_provision_approval}}

### CODE REVIEW

File: cfme/utils/appliance/implementations/ui.py [hunk 1]
```diff
@@ -172,6 +172,9 @@ def _check():
             result = self.browser.execute_script(self.ENSURE_PAGE_SAFE, silent=True)
             # TODO: Logging
             return bool(result)
+        time.sleep(0.2)
+        # added this sleep as loading, and other elements that we are checking
+        # in ENSURE_PAGE_SAFE script appears on page after script is run
```
Comment (psav) [inline cfme/utils/appliance/implementations/ui.py]: **REQUIRED:** I don't see why this is needed, we have silent failures on here and wait for 2 seconds overall? Any failure will jsut cause a retry in 0.2 seconds.

Comment (rlbabyuk) [reply cfme/utils/appliance/implementations/ui.py]: removed

File: cfme/common/vm_views.py [hunk 2]
```diff
@@ -373,6 +373,11 @@ class ProvisionView(BaseLoggedInPage):
     breadcrumb = BreadCrumb()
     image_table = Table('//div[@id="pre_prov_div"]//table')
 
+    @View.nested
+    class sidebar(View):  # noqa
+        extend_button = Text(locator='//div[contains(@class, "resize-right")]')
```
Comment (psav) [inline cfme/common/vm_views.py]: **REQUIRED:** Please use either:
```extend / reduce```
or
```increase / decrease```

File: cfme/common/vm_views.py [hunk 3]
```diff
@@ -382,6 +387,14 @@ class form(BasicProvisionFormView):  # noqa
         submit_button = Button('Submit')  # Submit for 2nd page, tabular form
         cancel_button = Button('Cancel')
 
+        def _select_template(self, template_name, provider_name):
+            try:
+                row = self.parent.image_table.row(name=template_name, provider=provider_name)
+            except IndexError:
+                raise TemplateNotFound('Cannot find template "{}" for provider "{}"'
+                                       .format(template_name, provider_name))
+            row.click()
```
Comment (psav) [inline cfme/common/vm_views.py]: **REQUIRED:** How can we click if we hit an exception finding it?

File: cfme/common/vm_views.py (hunk 2 above)
Comment (rlbabyuk) [reply cfme/common/vm_views.py]: done

File: cfme/common/vm_views.py (hunk 3 above)
Comment (rlbabyuk) [reply cfme/common/vm_views.py]: done

**Pull request merged**